src/Simulation/CsharpGeneration.App/Program.fs

```diff
@@ -29,6 +28,10 @@ type Options = {
       HelpText = "Destination folder where the output of the compilation will be generated.")>]
     OutputFolder : string
 
+    [<Option("test", Required = false, Default = false, 
```

Comment:
Why is this an option? If it *has* to be an option, then why is there no way to turn it off from a user project?